src/containers/forms/OrganizationForm/index.tsx

```diff
@@ -0,0 +1,191 @@
+import { ErrorMessage, Field, Form, Formik } from 'formik';
+import moment from 'moment';
+import React, { useState } from 'react';
+import { Redirect } from 'react-router';
+import { Button, Label } from 'reactstrap';
+import { FormGroup } from 'reactstrap';
+import * as Yup from 'yup';
+import {
+  OPENSRP_ORGANIZATION_ENDPOINT,
+  ORGANIZATION_LABEL,
+  REQUIRED,
+  SAVE,
+  SAVING,
+  TEAM_LIST_URL,
+} from '../../../constants';
+import { generateNameSpacedUUID } from '../../../helpers/utils';
+import { OpenSRPService } from '../../../services/opensrp';
+
+const OrgFormNameSpace = '9a4c8cb0-df70-11e9-b38b-57f114a50538';
+
+/** default type value for organizations */
+const defaultOrganizationType = {
```

Comment:
not if it should be move or not, but have we considered that, is there another file to centralize this type of constant in? If we don't think it should move, that's fine, just wondering

Reply:
Moved this to the organization Ducks module, seems like it more naturally belongs there over a constants file.